cmd/internal/cli/build_linux.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 func run(cmd *cobra.Command, args []string) {
+
```

Comment:
Why the extra newline?

```suggestion
```

Reply:
Sorry. Not intentional. I was relying on vscode to format the extra lines but apparently it doesn't. Will clean up. Thanks for pointing out.

Reply:
Sure! No problem :+1: